modules/running-a-pipeline-in-jupyterlab.adoc

```diff
@@ -26,12 +26,18 @@ endif::[]
 . In the Pipeline Editor user interface, click *Run Pipeline* (image:images/jupyterlab-run-pipeline-button.png[The Runtimes icon]).
 +
 The *Run Pipeline* dialog appears. The *Pipeline Name* field is automatically populated with the pipeline file name.
++
+[NOTE]
+====
+After you run your pipeline, a pipeline experiment containing your pipeline run is automatically created on the *Experiments* -> *Experiments and runs* page in the {productname-short} dashboard. The experiment name matches the name that you assign to the pipeline. 
```

Comment:
Suggest using past tense 'assigned' here. (Assuming we're referring to the name given to the pipeline on import/creation.)

Reply:
Fixed, thanks.